src/app/Application.cpp

```diff
@@ -171,7 +177,7 @@ void Application::updateAvailable(ApplicationVersionDetails details) {
     msgBox.exec();
 
 #if WIN32
-    if (msgBox.clickedButton() == downloadNewVersionBtn) {
+    if (msgBox.clickedButton() == downloadNewVersionBtn.get()) {
```

Comment:
I'd usually not allow this change as it makes 0 sense to be in this PR. But this is going to be changed anyways in #75, so I'll allow it.